third_party/winuwp_h264/H264Encoder/H264Encoder.h

```diff
@@ -56,16 +56,16 @@ class WinUWPH264EncoderImpl : public VideoEncoder, public IH264EncodingCallback
 
  private:
   ComPtr<IMFSample> FromVideoFrame(const VideoFrame& frame);
-  int InitEncoderWithSettings(const VideoCodec* codec_settings);
+  int InitEncoderWithSettings();
+  int ReleaseWriter();
+  LONGLONG GetFrameTimestampHns(const VideoFrame& frame) const;
```

Comment:
Document that "Hns" means "hundreds of nanoseconds"? (the standard Media Foundation timestamp unit)

Reply:
TBH I didn't know what it meant, I just copied the existing names 😄 Will do.

Reply:
I suspected as much. I would never have guessed without reading the RFC of RTP while looking for something else.